tests/manage/mcg/ui/test_mcg_ui.py

```diff
@@ -298,6 +304,9 @@ def test_obc_creation_and_deletion(
             obc_bucketclass == bucketclass
         ), f"BucketClass mismatch. Expected: {bucketclass}, found: {obc_bucketclass}"
 
+        if verify_ob_removal and obc_ui_obj.ocp_version_semantic > version.VERSION_4_12:
```

Comment:
Any reason for checking with the OCP version here? the issue was fixed with ODF 4.12.0

Reply:
my bad - it should be:

if verify_ob_removal and obc_ui_obj.ocp_version_semantic > version.VERSION_4_11:

Bug #2097772  was addressed in ODF 4.12

Reply:
@DanielOsypenko Looks like this comment is not yet addressed, please check

Reply:
yes, updated